riscv/v_ext_macros.h

```diff
@@ -1802,15 +1802,15 @@ reg_t index[P.VU.vlmax]; \
     case e16: { \
       float32_t &vd = P.VU.elt<float32_t>(rd_num, i, true); \
       float32_t vs2 = f16_to_f32(P.VU.elt<float16_t>(rs2_num, i)); \
-      float32_t rs1 = f16_to_f32(FRS1_H)); \
+      float32_t rs1 = f16_to_f32(FRS1_H); \
       BODY16; \
       set_fp_exceptions; \
       break; \
     } \
     case e32: { \
       float64_t &vd = P.VU.elt<float64_t>(rd_num, i, true); \
       float64_t vs2 = f32_to_f64(P.VU.elt<float32_t>(rs2_num, i)); \
-      float64_t rs1 = f32_to_f64(FRS1_F)); \
+      float64_t rs1 = f32_to_f64(FRS1_F); \
```

Comment:
I don't understand the changes to this file. This doesn't seem to have anything to do with the rest of this commit,

Reply:
Sorry, I fixed this mistake, which was introduced when I tried to rebase to upstream, in the wrong commit. I'll update this later.